dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -170,17 +164,24 @@ private ZonedIntervalIterable(
         "Start date %s must precede end date %s",
         queryLogStart,
         queryLogEnd);
-    this.start = queryLogStart.truncatedTo(unit);
-    this.end = queryLogEnd.truncatedTo(unit);
 
-    if (!start.equals(queryLogStart)) {
-      LOG.warn("Start time has been truncated to {}", start);
-    }
+    this.start = queryLogStart.truncatedTo(ChronoUnit.SECONDS);
```

Comment:
`queryLogStart` was previously truncated to the unit. Should it now be truncated to the duration instead of `SECONDS`?

Reply:
As per our discussion, we should have several methods of truncating `start` & `end`. The connector should be able to take their own decision on which one to use.